packages/core/src/command/consolidated/consolidated-create.ts

```diff
@@ -35,20 +35,72 @@ export async function createConsolidatedFromConversions({
   patientId,
   destinationBucketName = getConsolidatedLocation(),
   sourceBucketName = getConsolidatedSourceLocation(),
-  logMemUsage,
 }: ConsolidatePatientDataCommand): Promise<Bundle> {
   const { log } = out(`createConsolidatedFromConversions - cx ${cxId}, pat ${patientId}`);
 
-  const consolidated = buildConsolidatedBundle();
+  const [conversions, docRefs] = await Promise.all([
+    getConversions({ cxId, patientId, sourceBucketName }),
+    getDocumentReferences({ cxId, patientId }),
+  ]);
+  log(`Got ${conversions} resources from conversions`);
 
-  const inputBundles = await listConversionBundles({ cxId, patientId, sourceBucketName, log });
-  if (!inputBundles || inputBundles.length < 1) {
+  const withDups = buildConsolidatedBundle();
+  withDups.entry = [...conversions, ...docRefs.map(buildBundleEntry)];
+  withDups.total = withDups.entry.length;
+  log(`Added ${docRefs.length} docRefs, to a total of ${withDups.entry.length} entries`);
+
+  log(`Deduplicating consolidated bundle...`);
+  const deduped = deduplicate({ cxId, patientId, bundle: withDups });
+  log(`...done, from ${withDups.entry?.length} to ${deduped.entry?.length} resources`);
+
+  const dedupDestFileName = createConsolidatedDataFilePath(cxId, patientId, true);
+  const withDupsDestFileName = createConsolidatedDataFilePath(cxId, patientId, false);
+  log(`Storing consolidated bundle on ${destinationBucketName}, key ${dedupDestFileName}`);
+  log(`Storing consolidated bundle w/ dups on ${destinationBucketName}, key ${dedupDestFileName}`);
+  await Promise.all([
+    s3Utils.uploadFile({
+      bucket: destinationBucketName,
+      key: dedupDestFileName,
+      file: Buffer.from(JSON.stringify(deduped)),
+      contentType: "application/json",
+    }),
+    s3Utils.uploadFile({
+      bucket: destinationBucketName,
+      key: withDupsDestFileName,
+      file: Buffer.from(JSON.stringify(withDups)),
+      contentType: "application/json",
+    }),
+  ]);
+
+  log(`Done`);
+  return deduped;
```

Comment:
Would we ever want to return non-deduped?

Reply:
were not and don't think we will - but easy to adjust if needed later IMO